### PR TITLE
Show user polls on account page

### PR DESCRIPTION
**Pull request by elsmr**
- [x] everything
- [x] write tests

### CODE REVIEW

File: src/main/java/com/pollistics/controllers/PollController.java [hunk 1]
```diff
@@ -50,13 +51,12 @@ public String pollDetail(@PathVariable String pollId, Model model, HttpServletRe
...
 	@PostMapping(value = "/polls/create")
 	public String createPoll(HttpServletRequest request, @Valid @ModelAttribute("poll") Poll poll, BindingResult result, Principal principal, Model model) {
-		HashMap<String, Integer> options = new HashMap<>();
-		int i = 0;
-		while (!request.getParameter("option" + i).trim().isEmpty()) {
-			String option = request.getParameter("option" + i);
-			options.put(option, 0);
-			i++;
-		}
+		HashMap<String, Integer> options =
```
Comment (Haroenv) [inline src/main/java/com/pollistics/controllers/PollController.java:54]: ma ja gast fancy

File: src/test/java/com/pollistics/controllers/PollControllerTests.java [hunk 2]
```diff
@@ -90,7 +90,7 @@ public void createValidPollTest() {
 		}
 	}
 	
-	/*@Test
```
Comment (Haroenv) [inline src/test/java/com/pollistics/controllers/PollControllerTests.java:93]: top kek

File: src/main/java/com/pollistics/controllers/PollController.java (hunk 1 above)
Comment (elsmr) [reply src/main/java/com/pollistics/controllers/PollController.java:54]: :scream:

**Review by Haroenv — APPROVED** [review]
the moeit worth

**Pull request merged**